pkg/server/methods/executeCommand.go

```diff
@@ -14,13 +14,16 @@ func (methods *Methods) ExecuteCommand(reply jsonrpc2.Replier, req jsonrpc2.Requ
 		return reply(methods.Ctx, nil, fmt.Errorf("%s: %w", jsonrpc2.ErrParse, err))
 	}
 	if params.Command == "setToken" {
+		token := params.Arguments[0].(string)
+		if methods.Cache.TokenCache.GetToken() != token {
+			methods.Cache.OrbCache.RemoveOrbs()
+			methods.Cache.RemoveOrbFiles()
```

Comment:
I think we should first remove the files then remove the orbs, right ?
otherwise we'll remove first the orbs from the cache and then we have nothing left in the cache about orbs, and we'll never delete the source files from the user disk since orbs will not be in the cache anymore